docs/ref/settings.txt

```diff
@@ -146,7 +146,7 @@ The :setting:`APPEND_SLASH` setting is only used if
 CACHES
 ------
 
-Default::
```

Comment:
This suggestion relates mostly to document-wide consistency, however initially it was spotted because of the use of a  '::' double-colon in the range function in Python.  It is okay, i believe, to use a double colon (i.e, Default::) for every example that has a multi-line code block which follows it, and to use a single colon (i.e., Default:) for those examples which are shown on a single line, but it would be better to have a uniform pattern throughout the document.